robusta_krr/strategies/simple.py

```diff
@@ -48,11 +48,24 @@ class SimpleStrategy(BaseStrategy[SimpleStrategySettings]):
     __display_name__ = "simple"
     __rich_console__ = True
 
-    def run(self, history_data: HistoryData, object_data: K8sObjectData) -> RunResult:
+    def __calculate_cpu_proposal(self, history_data: HistoryData, object_data: K8sObjectData) -> ResourceRecommendation:
+        if object_data.hpa is not None and object_data.hpa.target_cpu_utilization_percentage is not None:
+            return ResourceRecommendation.undefined()
```

Comment:
for the table for OOMKill it was requested to add a star by the reccomendation and at the bottom add a line like "*this container had oomkills"
I would consult tomer if something simmilar should be done for HPA
like saying "**this is run by HPA so the metrics can not be calculated"

Reply:
Created a PR for the playbook also
https://github.com/robusta-dev/robusta/pull/953